sdk/tests/conformance2/reading/read-pixels-pack-parameters.html

```diff
@@ -56,6 +65,45 @@
     }
 }
 
+function paintWebGLCanvas(gl)
+{
+    var canvas2d = document.getElementById("canvas2d");
+    var ctx2d = canvas2d.getContext("2d");
+    for (var row = 0; row < 4; ++row) {
+        ctx2d.fillStyle = "rgba(" + expectedColor[row][0] + ", " +
+            expectedColor[row][1] + ", " + expectedColor[row][2] + ", " +
+            expectedColor[row][3] + ")";
+        ctx2d.fillRect(0, row, 4, 1);
```

Comment:
This way of setting up this 2D texture seems to test more code paths than desired. We've been discussing the exact color spaces that are used for 2D canvas, WebGL, etc., and they might not match. A better way would just be to create a Uint8Array and fill it with the data. Just allocating a 1x4 texture should be sufficient. Could you do that instead?